plugin/src/main/resources/META-INF/fabric8/profiles-default.yml

```diff
@@ -61,8 +61,8 @@
   enricher:
     includes:
     - fmp-name
-    - fmp-image
     - fmp-controller
+    - fmp-image
```

Comment:
good point :) guess we should move also fmp-service up, since fmp-controller and fmp-service are both enrichers which 'create' objects. That will be not that critical anymore when #678 is in place.